config/opal_check_libfabric.m4

```diff
@@ -26,12 +26,26 @@ AC_DEFUN([OPAL_CHECK_LIBFABRIC],[
 	# Add --with options
 	AC_ARG_WITH([libfabric],
 		    [AC_HELP_STRING([--with-libfabric=DIR],
-				    [Specify location of libfabric installation, adding DIR/include to the default search location for libfabric headers, and DIR/lib or DIR/lib64 to the default search location for libfabric libraries.  Error if libfabric support cannot be found.])])
+				    [Specify location of OFI libfabric installation, adding DIR/include to the default search location for libfabric headers, and DIR/lib or DIR/lib64 to the default search location for libfabric libraries.  Error if libfabric support cannot be found.])])
 	AC_ARG_WITH([libfabric-libdir],
```

Comment:
Would it be better to flag these options as "deprecated in favor of --with-ofi", and put the detailed help message over there?